### DISCUSSION TITLE
Can I use Seurat to analyze MARS-Seq data?

### DISCUSSION THREAD

**Discussion by Sahu-Mousumi** [satijalab/seurat#6091] [Q&A]
Hi,
I have matrix files from MARS-Seq data obtained from GEO. Can I use standard 10X genome data integration pipeline to analyze these? 
Any suggestion will be appreciated!

Thank you!

**Comment by saketkc** [comment]
Yes, you can use Seurat to analyze MARS-seq (or any UMI based data). An example of fetching files from GEO and processing them directly is [here](https://github.com/saketkc/scRNA_NB_comparison/blob/master/analysis/03A_Ding-Mixture-HEK-3T3.Rmd#L67)
